sources/engine/Xenko.UI/UIElement.Events.cs

```diff
@@ -98,6 +98,9 @@ internal set
             }
         }
 
+        [DataMemberIgnore]
+        public bool RequiresMouseOverUpdate { get; set; }
```

Comment:
Is this field useful to users ? If not you could change it to internal. If it is useful could you add a summary to it ?

Reply:
Added summary